test/test_fx.py

```diff
@@ -37,6 +37,11 @@ def forward(self, x):
 def a_non_torch_leaf(a, b):
     return a + b
 
+def a_lifted_leaf(a, b):
+    return a[0] + a[1] + b
+
+wrap('a_lifted_leaf')
```

Comment:
Note: we can also allow @wrap on functions to still work. In the use where the argument to wrap is a function rather than a string, simple get the name from `__code__.co_name`. Then you can just write it as an annotation as before.